assets/js/modules/analytics-4/datastore/enhanced-measurement.js

```diff
@@ -427,6 +442,46 @@ const baseSelectors = {
 
 		return ! isEqual( settings, savedSettings );
 	},
+
+	/**
+	 * This is a utility selector that checks if either the GA4 module settings, or the
+	 * current web data stream's enhanced measurement settings have changed.
```

Comment:
```suggestion
	 * This is a utility selector that checks if either the GA4 module settings, or the
	 * current web data stream's enhanced measurement settings, have changed.
```

Reply:
Ah, the old Oxford comma :)

I believe this is an optional grammatical construct, but I'm entirely happy to use it here :smile: